### PR TITLE
Figure.wiggle: Remove parameter 'columns', use 'incols' instead.

### PR DESCRIPTION
**Pull request by seisman**
**Description of proposed changes**

Related to https://github.com/GenericMappingTools/pygmt/issues/1966.

**Reminders**

- [ ] Run `make format` and `make check` to make sure the code follows the style guide.
- [ ] Add tests for new features or tests that would have caught the bug that you're fixing.
- [ ] Add new public functions/methods/classes to `doc/api/index.rst`.
- [ ] Write detailed docstrings for all functions/methods.
- [ ] If wrapping a new module, open a 'Wrap new GMT module' issue and submit reasonably-sized PRs.
- [ ] If adding new functionality, add an example to docstrings or tutorials.

**Slash Commands**

You can write slash commands (`/command`) in the first line of a comment to perform
specific operations. Supported slash commands are:

- `/format`: automatically format and lint the code
- `/test-gmt-dev`: run full tests on the latest GMT development version

### CODE REVIEW

File: pygmt/tests/test_wiggle.py [hunk 1]
```diff
@@ -32,12 +32,9 @@ def test_wiggle():
...
 
 @pytest.mark.mpl_image_compare(filename="test_wiggle.png")
-def test_wiggle_deprecate_columns_to_incols():
+def test_wiggle_data_incols():
     """
-    Make sure that the old parameter "columns" is supported and it reports a
-    warning.
-
-    Modified from the test_wiggle() test.
+    Make sure that incols works with input data array.
```
Comment (weiji14) [inline pygmt/tests/test_wiggle.py:37]:
```suggestion
    Make sure that incols works with input data array and incols parameter.
```

File: pygmt/tests/test_wiggle.py [hunk 2]
```diff
@@ -32,12 +32,9 @@ def test_wiggle():
...
     """
 
     # put data into numpy array and swap x and y columns
```
Comment (weiji14) [inline pygmt/tests/test_wiggle.py:40]: Line41. Need to change the 'columns' to 'incols'

File: pygmt/tests/test_wiggle.py (hunk 1 above)
Comment (seisman) [reply pygmt/tests/test_wiggle.py:37]: Do you mean:
```
 Make sure that incols parameter works with input data array.
```

Comment (weiji14) [reply pygmt/tests/test_wiggle.py:37]: Ah yes, let me change that.

**Review by weiji14 — APPROVED** [review]

**Pull request merged**